modules/exploits/multi/php/wordpress_duplicator.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
```

Comment:
Add a comment after this mentioning its Manual due to the manipulation of the config file breaking the site